src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,59 @@
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+
+  border: 1vmin #000;
+  border-style: dotted;
+  border-radius: 50%;
+
+  height: 80vmin;
+  width: 80vmin;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__minutes {
+    position: absolute;
+    bottom: 50%;
+
+    height: 20vmin;
+    width: 3vmin;
+
+    background-color: #0700ff;
+    transform-origin: bottom center;
+
+    animation: rotation 3600s steps(60) infinite;
+
+    @include rotation-clock;

```

Comment:
Ensure that the `rotation-clock` mixin is defined in your SCSS files. If it's missing, the styles relying on it won't work as expected.